src/components/Searchbar/InputField.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+const InputField = ({ searchInput, handleChange }) => {
+  return (
+    <>
```

Comment:
Here we don't need React.Fragment, because we only have one element.

Reply:
dziękuje to dobra uwaga